templates/layout/_topbar.html.twig

```diff
@@ -4,13 +4,13 @@
             <a class="stretched-link" href="{{ path('front') }}">{{ 'all'|trans }}</a>
         </li>
         <li class="{{ html_classes({'active': is_route_name_contains('_subscribed')}) }}">
-            <a class="stretched-link" href="{{ path('front_subscribed') }}">{{ 'subscribed'|trans }}</a>
+            <a class="stretched-link" href="{{ path('front') }}">{{ 'subscribed'|trans }}</a>
         </li>
         <li class="{{ html_classes({'active': is_route_name_contains('_moderated')}) }}">
-            <a class="stretched-link" href="{{ path('front_moderated') }}">{{ 'moderated'|trans }}</a>
+            <a class="stretched-link" href="{{ path('front') }}">{{ 'moderated'|trans }}</a>
         </li>
         <li class="{{ html_classes({'active': is_route_name_contains('_favourite')}) }}">
-            <a class="stretched-link" href="{{ path('front_favourite') }}">{{ 'favourites'|trans }}</a>
+            <a class="stretched-link" href="{{ path('front') }}">{{ 'favourites'|trans }}</a>
         </li>
```

Comment:
this is also not working. Neigher the "active" part nor the link

Reply:
Oooh... okay, I completely misunderstood this whole part. I don't have my top bar turned on, and I thought this was just something else. Yeah I'd just completely broken it. I made what I think should be a fix.